src/preferences/screens/Preferences.tsx

```diff
@@ -1,73 +1,107 @@
-import {Paper} from "@mui/material";
-import {ConnectingAirports, Explore, Flight, Hotel, KingBed, MeetingRoom} from "@mui/icons-material";
+import { Paper } from "@mui/material";
+import React, { useEffect, useState } from "react";
+import { ConnectingAirports, Explore, Hotel, MeetingRoom } from "@mui/icons-material";
+
+// Define the Reservation type
+type Reservation = {
+    hotelName: string;
+    roomNames: string[];
+    locationFrom: string;
+    locationTo: string;
```

Comment:
nazwy zmiennych brzmią jakby były typu `Location`, a to zwykły string - a lokacja jest bardziej złożonym typem

Reply:
Uwzględnione!